Obsidian.API/_Interfaces/IPlayer.cs

```diff
@@ -51,7 +51,9 @@ public interface IPlayer : ILiving
         public Task SendNamedSoundAsync(string name, SoundPosition position, SoundCategory category = SoundCategory.Master, float pitch = 1f, float volume = 1f);
         public Task KickAsync(IChatMessage reason);
         public Task KickAsync(string reason);
-
+        public Task<bool> GrantPermission(string permission);
+        public Task<bool> RevokePermission(string permission);
+        public Task<bool> HavePermission(string permission);
```

Comment:
Change the name of the method to **HasPermission** ?

Reply:
Okeh